include/vcpkg/base/messages.h

```diff
@@ -72,6 +72,15 @@ namespace vcpkg::msg
 
 namespace vcpkg
 {
+    // constants for the
+    // <file>:line:col: <prefix>: <content>
+    // error message format
+    inline constexpr StringLiteral ErrorPrefix = "error: ";
+    inline constexpr StringLiteral InternalErrorPrefix = "internal error: ";
+    inline constexpr StringLiteral MessagePrefix = "message: ";
```

Comment:
How is this different than `note: `?

Reply:
`note:` means 'stapled to some other error or warning'. `message:` is its own thing. The only case I know of right now is the 'versions are ok' message for `x-ci-verify-versions --verbose` mode.